pages/governance.md

```diff
@@ -0,0 +1,291 @@
+JuMP project.
+
+### Financial Decisions
+
+All financial decisions are made by the [Steering Committee] to ensure any funds
+are spent in a manner that furthers the [mission](#mission) of JuMP. Financial
+decisions require majority approval by [Steering Committee] members.
+
+Community members proposing decisions with a financial aspect should contact the
+[Steering Committee] directly, or table their proposal as an agenda item for
+discussion on a [monthly developer call].
+
+### Non-financial Decisions
+
+All non-financial decisions are made via consensus of the [core contributors]
+and relevant [repository maintainers]. [Emeritus core contributors](#emeritus-core-contributors)
+are not considered to be [core contributors] for this purpose.
+
+Code-related decisions, such when a pull request is ready to be accepted and
+merged, should be discussed via the relevant Github issues and pull requests. If
+consensus cannot be achieved, the community member proposing the change may be
+invited by a [core contributor](#core-contributors) to present their proposal at
+a [monthly developer call] for further discussion and community input.
+
+Non-code-related decisions, such as long-term strategic planning for JuMP,
+should either be discussed in a Github issue, or tabled as an agenda item and
+discussed on a [monthly developer call].
+
+If consensus on a non-financial decision cannot be achieved, the final decision
+will be made by the [BDFL](#benevolent-dictator-for-life).
+
+The [Steering Committee] can gain additional decision-making power if the [core
+contributors] decide to delegate.
+
+### Conflict of Interest
+
+It is expected that community members will be employed at a wide range of
+companies, universities and non-profit organizations. Because of this, it is
+possible that members will have conflicts of interest. Such conflicts of
+interest include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work,
+  outside of JuMP that may influence their work on JuMP.
+* Access to proprietary information of their employer that could potentially
+  leak into their work with JuMP.
+
+All members of the [Steering Committee] shall disclose to the [Steering
+Committee] any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in [Steering Committee]
+discussions on that issue, but must recuse themselves from voting.
+
+[Core contributors](#core-contributors) and [repository maintainers] should also
+disclose conflicts of interest with other [core contributors] and step back from
+decisions when conflicts of interests are in play.
+
+## Github Permissions
+
+[Github permissions][permissions] are used to control access to repositories in
+the [JuMP-dev Github organization].
+
+Anyone with commit access to a repository is trusted to use it in a way that is
+consistent with the [non-financial decisions](#non-financial-decisions) process.
```

Comment:
It's fine to say "consistent with the Decision Making Process", because commit access might be used for financial-related decisions also.